app/controllers/users_controller.rb

```diff
@@ -8,6 +8,7 @@ def create
     @user = User.new(user_params)
     if @user.save
       session[:user_id] = @user.id
+      log_in @user
```

Comment:
It seems like this here and the `log_in` function in `ApplicationController` are the only two things that should really be in this PR for redirecting after account creation. Do you think you could possibly muddle through the git history so these are the only things included in the PR? I'm just a bit cautious approving this PR, because it contains so much of the code for ATH-1116.

The other solution is that you put out the PR for ATH-1116 first, I review that and we merge it, then the only changes that turn up here should be the redirecting after account creation.

Reply:
The testing for login are also included in this PR, which was intended. 

I can try to remove the logout functionality though. It looks as though the sequencing of the story IDs confused me and all work was done on this branch.

Reply:
ATH-1117 should be independent of ATH-1116 now

Reply:
Awesome! Taking a look now.